src/host/ut_host/Host.UnitTests.vcxproj

```diff
@@ -35,7 +35,6 @@
     <ClCompile Include="UtilsTests.cpp" />
     <ClCompile Include="Utf8ToWideCharParserTests.cpp" />
     <ClCompile Include="InputBufferTests.cpp" />
-    <ClCompile Include="ReadWaitTests.cpp" />

```

Comment:
remove from sources

Reply:
Seems like it never was part of any sources file?